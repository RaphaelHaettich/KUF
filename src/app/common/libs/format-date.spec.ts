import formatDate from './format-date';

describe('formatDate', () => {
    it('should return an empty string if the input is empty', () => {
        expect(formatDate(undefined)).toBe('');
    });

    it('should format the date correctly', () => {
        const unixTimeStampString = 1625097600000;
        const formattedDate = formatDate(unixTimeStampString, 'de');
        expect(formattedDate).toMatchSnapshot();
    });

    it('should format the date correctly for a different language', () => {
        const unixTimeStampString = 1625097600000;
        const formattedDate = formatDate(unixTimeStampString, 'fr');
        expect(formattedDate).toMatchSnapshot();
    });
});
