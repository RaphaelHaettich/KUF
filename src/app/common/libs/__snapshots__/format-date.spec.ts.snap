// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`formatDate should format the date correctly 1`] = `"01.07.2021"`;

exports[`formatDate should format the date correctly for a different language 1`] = `"01.07.2021"`;
